store/rootmulti/store.go

```diff
@@ -598,9 +601,11 @@ func (rs *Store) PruneStores(clearPruningManager bool, pruningHeights []int64) (
 		return nil
 	}
 
-	rs.logger.Debug("pruning heights", "heights", pruningHeights)
+	rs.logger.Debug("prune store", "heights", pruningHeights)
 
 	for key, store := range rs.stores {
+		rs.logger.Debug("prune store", "key", key)
```

Comment:
Let's add the store as a field to the logger too

Reply:
Saw the suggestion to log "name', store.Name, but store.name is a private lower case variable.